### ISSUE TITLE
django-tui unresponsive on Python 3.13 on MacOS (iTerm2)

### ISSUE THREAD

**Issue by anze3db** [anze3db/django-tui#8]
django tui is unresponsive when started on Python 3.13 on MacOS (iTerm2).

The only hint is these characters that show up when pressing keys:

<img width="2384" height="3054" alt="Image" src="https://github.com/user-attachments/assets/5b7548d4-448a-4091-b462-53c158cc553c" />

Interestingly, if I SSH to a Linux machine and start Django tui there there are no issues.

I couldn't fully figure it out but I believe there is a warning message that gets printed to the screen and messes up the rendering.

**Comment by Rahulbeniwal26119** [comment]
Hi @anze3db 
Is it still open? If yes then i would like to check against latest Python Version

**Comment by anze3db** [comment]
Yes, this is still open. Although I have found out that it's only broken with Python 3.13 on iTerm2 on MacOS. If I SSH to a Linux machine and open it there it works.

**Comment by Rahulbeniwal26119** [comment]
Sure, i will start working on it.

Thank You
